Services/Concurrency/RatedCounter.cs

```diff
@@ -177,21 +177,40 @@ public long GetPause()
             return false;
         }
 
+        public double GetThroughputForMessages()
```

Comment:
This method can be removed. GetMessagesPerSecond will return 0 if this.timestamps.count is 0.